todo_app/app.py

```diff
@@ -1,11 +1,30 @@
-from flask import Flask
+from flask import Flask, request, render_template, redirect, abort
+from todo_app.data.todoItemManager import todo_list
```

Comment:
No problem at all with this alternative, I'm only afraid you put in some extra work for something we'll be disposing of very shortly! 😄